pom.xml

```diff
@@ -67,15 +67,15 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-api</artifactId>
-			<version>5.5.1</version>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-module-junit4</artifactId>
```

Comment:
Note: We should try to use junit5 as soon as possible.

Reply:
Not possible with PowerMockito